app/extensions/users-permissions/config/policies/rateLimit.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const lazyRateLimit = {
+  get RateLimit() {
+    return require('koa2-ratelimit').RateLimit;
+  },
+};
+
+module.exports = async (ctx, next) => {
+  const message = [
+    {
+      messages: [
+        {
+          id: 'Auth.form.error.ratelimit',
+          message: 'Too many attempts, please try again in a minute.',
+        },
+      ],
+    },
+  ];
+
+  return lazyRateLimit.RateLimit.middleware(
+    Object.assign(
+      {},
+      {
+        interval: 1 * 60 * 1000,
+        max: 999999,
```

Comment:
Change back to limit 5. and I keep the code to let us know that there is a rate-limit policy applied to user-permissions.
```suggestion
        max: 5,
```